src/components/BackendExportEntitiesForm.tsx

```diff
@@ -0,0 +1,257 @@
+      color: theme.palette.text.primary
+    },
+    paginationLink: {
+      fontFamily: theme.typography.fontFamily,
+      fontSize: "17px",
+      marginTop: "-4px",
+      letterSpacing: "normal",
+    }
+  };
+});
+
+export default function BackendExportEntitiesForm() {
+  const opslevelPluginApi = useApi(opslevelPluginApiRef);
+  const [configuration, setConfiguration] = useState<AutoSyncConfiguration | null>(null);
+  const [configurationSaving, setConfigurationSaving] = useState<boolean>(false);
+  const [dirty, setDirty] = useState<boolean>(false);
+  const [currentRun, setCurrentRun] = useState<AutoSyncExecution | null>(null);
+  const [currentRunIndex, setCurrentRunIndex] = useState(0);
+  const [runCount, setRunCount] = useState<number | null>(null);
+  const [configExpanded, setConfigExpanded] = useState(false);
+
+  const outputRef = useRef<any>();
+  const currentIndexRef = useRef<number>();
+  useEffect(() => { currentIndexRef.current = currentRunIndex }, [currentRunIndex]);
+  const currentRunRef = useRef<AutoSyncExecution | null>();
+  useEffect(() => { currentRunRef.current = currentRun }, [currentRun]);
+
+  const setSchedule = (schedule: string) => {
+    if(configuration === null) return;
+    if(schedule !== configuration.auto_sync_schedule) {
+      setConfiguration({ auto_sync_enabled: configuration.auto_sync_enabled, auto_sync_schedule: schedule });
+      setDirty(true);
+    }
+  };
+
+  const setEnabled = (enabled: boolean) => {
+    if(configuration === null) return;
+    setConfiguration({ auto_sync_enabled: enabled, auto_sync_schedule: configuration.auto_sync_schedule });
+    setDirty(true);
+  };
+
+  const loadRun = (index = currentRunIndex) => {
+    opslevelPluginApi.getAutoSyncExecution(index)
+      .then((ret) => {
+        const contentChanged = ret.rows.length > 0 && ret.rows[0].output !== currentRunRef?.current?.output;
+        setCurrentRun(ret.rows.length > 0 ? ret.rows[0] : null);
+        setRunCount(parseInt(ret.total_count, 10));
+        setCurrentRunIndex(index);
+        if (contentChanged) {
+          outputRef.current?.scroll?.({ top: outputRef.current.scrollHeight, behavior: 'smooth' });
+        }
+      });
+  };
+
+  const next = () => {
+    setCurrentRun(null);
+    setCurrentRunIndex(currentRunIndex + 1);
+    loadRun(currentRunIndex + 1);
+  };
+
+  const prev = () => {
+    setCurrentRun(null);
+    setCurrentRunIndex(currentRunIndex - 1);
+    loadRun(currentRunIndex - 1);
+  };
+
+  const saveConfiguration = () => {
+    if(configuration === null) return;
+    setConfigurationSaving(true);
+    opslevelPluginApi.setAutoSyncConfiguration(configuration)
+      .then((success) => {
+        setDirty(!success);
+        setConfigurationSaving(false);
+        setConfigExpanded(!success);
+      });
+  };
+
+  useEffect(() => {
+    opslevelPluginApi.getAutoSyncConfiguration()
+      .then((response) => setConfiguration(response));
+  }, []);
+
+  useEffect(loadRun, []);
```

Comment:
Some `useEffect` calls are missing deps. For example

```suggestion
  useEffect(loadRun, [loadRun]);
```

Reply:
discussing in other thread